src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -80,6 +80,39 @@ class ov::genai::VLMPipeline::VLMPipelineImpl {
         }
```

Comment:
BTW, I don't see that `m_generation_config` is read from disk here.

Reply:
Right, in current VLM pipeline implementation `m_generation_config` is not read from disk. I suppose it can be set now only with `set_generation_config()` method as shown in `src/README.md` sample.
Or I am missing something and didn't get your comment right?

Reply:
miniCPM's config is almost empty, I guess that's why it was forgotten. https://huggingface.co/openbmb/MiniCPM-V-2_6/blob/main/generation_config.json

Reply:
ok, than we need to add reading? Experience should be the same as with LLMPipeline